deploy-manage/deploy/elastic-cloud/serverless.md

```diff
@@ -61,38 +61,37 @@ Serverless projects run on cloud platforms, which may undergo changes in availab
 
 To learn more about serverless status, see [Service status](../../cloud-organization/service-status.md).
 
-## Answers to common serverless questions [general-what-is-serverless-elastic-answers-to-common-serverless-questions]
+## Frequently Asked Questions (FAQ) about Elastic Serverless Projects [general-what-is-serverless-elastic-answers-to-common-serverless-questions]
 
-**Is there migration support between hosted deployments and serverless projects?**
+The following FAQ addresses common questions about using Elastic serverless projects.
 
-Migration paths between hosted deployments and serverless projects are currently unsupported.
+### Pricing and availability
+**Q: Where can I learn about pricing for serverless?**  
+A: See serverless pricing information for [Search](https://www.elastic.co/pricing/serverless-search), [Observability](https://www.elastic.co/pricing/serverless-observability), and [Security](https://www.elastic.co/pricing/serverless-security).
 
-**How can I move data to or from serverless projects?**
+**Q: In which cloud regions is Elastic’s serverless offering available?**  
+A: Elastic serverless projects are available in select AWS regions, with plans to expand to more regions and cloud providers. For the latest list, refer to the [Serverless regions](/deploy-manage/deploy/elastic-cloud/regions.md).
```

Comment:
no strong opinion about "For the latest list" so will defer to Kaarina. empty link title will adopt the title of the target page 🪄 

```suggestion
A: {{serverless-full}} is available in select AWS, GCP, and Azure regions, with plans to expand to more regions. For more information, refer to [](/deploy-manage/deploy/elastic-cloud/regions.md).
```